lasagne/layers/special.py

```diff
@@ -376,6 +376,12 @@ class TransformerLayer(MergeLayer):
         output image (in both spatial dimensions). A value of 1 will keep the
         original size of the input. Values larger than 1 will downsample the
         input. Values below 1 will upsample the input.
+    border_mode : 'nearest', 'mirror', or 'wrap'
+        Determines how border conditions are handled during interpolation.  If
+        'nearest', points outside the grid are clipped to the boundary. If
+        'mirror', points are mirrored across the boundary. If 'wrap', points
+        wrap around to the other side of the grid.  See
+        http://stackoverflow.com/q/22669252/22670830#22670830 for details.
```

Comment:
Not sure if we want to refer to stackoverflow there... but if we do, we could use a shorter link: `See http://stackoverflow.com/a/22670830 for an illustration.`